packages/fluentui/react-northstar/src/components/Form/utils/formFieldBase.tsx

```diff
@@ -104,6 +104,7 @@ const _FormFieldBase = compose<'div', FormFieldBaseProps, {}, {}, {}>(
             getA11yProps('control', {
               error: !!errorMessage || null,
               ref,
+              ...(labelId && { labelId: labelId.current }),
```

Comment:
@layershifter We need this here because the `formFieldBehavior` will add `aria-labelledby="labelID messageID"` and since in the input the label now is being created inside I need to pass it down to the input so it adds the proper `id` to the label. So in the `Input` now it's a handled props but it will throw for the other components such as `Form.Dropdown`... 😞

Any ideas?

Reply:
I am thinking if I should use context for only this